mod/videos.php

```diff
@@ -150,25 +116,16 @@ function videos_post(App $a)
 
 		$video_id = $_POST['id'];
 
-		$r = q("SELECT `id`  FROM `attach` WHERE `uid` = %d AND `id` = '%s' LIMIT 1",
-			intval(local_user()),
-			DBA::escape($video_id)
-		);
+		$r = Attach::exists(['id' => $video_id, 'uid' => local_user()]);
+		if ($r === true) {
```

Comment:
You don't need the intermediate variable here, you can put the `Attach::exists` call directly in the `if ()`.